src/xcode/ENA/ENA/Source/Services/DownloadedPackagesStore/V3/DownloadedPackagesStoreV3.swift

```diff
@@ -0,0 +1,106 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import FMDB
+
+protocol DownloadedPackagesStoreV3: AnyObject {
+
+	func open()
+	func close()
+
+	func markPackagesAsCheckedForExposures(_ fingerprints: [String]) throws
+	
+	func set(country: Country.ID, hour: Int, day: String, etag: String?, package: SAPDownloadedPackage?) throws
+	func set(country: Country.ID, day: String, etag: String?, package: SAPDownloadedPackage?) throws
+
+	/// Fetch key packages with a given ETag
+	/// - Parameter etag: The ETag to match or `nil` to fetch all packages that do not contain an ETag
+	/// - Returns: A list of matching key packages or `nil` if no matching packages were found
+	func packages(with etag: String?) -> [SAPDownloadedPackage]?
+
+	/// Fetch key packages with a given list of ETags
+	/// - Parameter etags: The ETag list to match
+	/// - Returns: A list of matching key packages or `nil` if no matching packages were found
+	func packages(with etags: [String]) -> [SAPDownloadedPackage]?
+
+	func package(for day: String, country: Country.ID) -> SAPDownloadedPackage?
+	func hourlyPackages(for day: String, country: Country.ID) -> [SAPDownloadedPackage]
+	func hourlyPackagesNotCheckedForExposure(for day: String, country: Country.ID) -> [SAPDownloadedPackage]
+	func allDays(country: Country.ID) -> [String] // 2020-05-30
+	func allDaysNotCheckedForExposure(country: Country.ID) -> [String] // 2020-05-30
```

Comment:
```suggestion
	func dailyPackagesNotCheckedForExposure(country: Country.ID) -> [String] // 2020-05-30
```

Reply:
Would not rename it, because it does not return packages, but the day strings.